server/doc/swdesign/README.md

```diff
@@ -45,8 +45,8 @@ The StartupStateLoader loads the initial startup state, parses it and pushes it
 The Communication Middleware is responsible for:
 
 * establishing the connections to multiple Ankaios Agents.
-* forwarding the ExecutionRequests and ExecutionStateUpdates from AnkaiosServer to the proper Ankaios Agents through the ExecutionCommandChannel.
-* forwarding the StateChangeRequests from connected Agents to the AnkaiosServer.
+* forwarding the ExecutionRequests and ExecutionStateUpdates from AnkaiosServer to the proper Ankaios Agents through the FromServerChannel.
+* forwarding the ToServer messages from connected Agents to the AnkaiosServer.
```

Comment:
Here we are writing `ToServer message`, but below `To Server message`.

Reply:
I fixed this with [b70ec51](https://github.com/eclipse-ankaios/ankaios/pull/152/commits/b70ec51802ae7fcd85e1bd779122c2b443a28b1f).